src/clr-angular/utils/drag-and-drop/interfaces/draggable-event.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * Copyright (c) 2016-2017 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+import {DragPosition} from "./client-position";
+import {Draggable} from "./draggable";
+
+export interface DraggableEvent {
+    draggable: Draggable;
+    dragPosition: DragPosition;
+    dragAndDropGroup: string;
```

Comment:
If the draggable belongs to more than one group, then this could be either a string or an array, right?

Reply:
Ah this one is a bit different. This is why using the same input name "dragAndDropGroup"  on  both `clrDraggable` and `clrDroppable` makes it confusing. 

It's an event originated from `clrDraggable` so the `dragAndDropGroup` is going to be always a single string.

Reply:
I'll get to that part at some point in the review, but draggables can also belong to more than one group. I suspected you didn't take this into account as I went through the other files.